angular/projects/spark-angular/src/lib/components/inputs/Search.stories.ts

```diff
@@ -23,44 +23,191 @@ export default {
   },
   decorators: [
     storyWrapper(
-      storyContent => (
+      (storyContent) =>
         `<div class="sprk-o-Box">
           <form (submit)="onSubmit($event)" #sampleForm="ngForm">
             ${storyContent}
           </form>
-        <div>`
-      )
-    )
+        <div>`,
+    ),
   ],
   props: {
     onSubmit(event): void {
       this.form_submitted = true;
-    }
+    },
   },
   parameters: {
     info: `
 ${markdownDocumentationLinkBuilder('input')}
     `,
     docs: { iframeHeight: 200 },
-  }
+  },
 };
 
 const modules = {
   imports: [
     SprkIconInputContainerModule,
-    SparkInputContainerModule,
+    SprkInputContainerModule,
     SprkLabelModule,
     SprkInputModule,
     SprkIconModule,
-    SprkFieldErrorModule
+    SprkFieldErrorModule,
   ],
 };
 
 export const searchInput = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-input-container>
+      <div class="sprk-b-InputContainer__icon-container">
+        <label
+          for="search"
+          hasIcon="true"
+          isHidden="true"
+          sprkLabel
+        >
+          Search
+        </label>
+        <sprk-icon
+          iconName="search"
+          additionalClasses="sprk-b-InlineSearch__icon sprk-c-Icon--filled-current-color sprk-c-Icon--stroke-current-color"
+          sprk-input-icon
+        ></sprk-icon>
+        <input
+          id="search"
+          name="inline_search_input"
+          class="sprk-b-TextInput--has-svg-icon"
+          type="search"
+          placeholder="Search"
+          [(ngModel)]="inline_search_input"
+          #inlineSearchInput="ngModel"
+          data-id="input-search-1"
```

Comment:
the sprkInput directive has an idString input we should use here.

Reply:
I'll take this and all the ones related to this!

Reply:
All of these comments are fixed in this PR. Also noticed places where we could use idString and added them.
https://github.com/sparkdesignsystem/spark-design-system/pull/3959